src/test/java/bookstore/config/CustomMySqlContainer.java

```diff
@@ -0,0 +1,32 @@
+package bookstore.config;
+import org.testcontainers.containers.MySQLContainer;
```

Comment:
consider configuring auto imports refactor in your IDE
```suggestion

import org.testcontainers.containers.MySQLContainer;
```